apps/studio/src/server/modules/site/site.router.ts

```diff
@@ -41,11 +41,12 @@ export const siteRouter = router({
       return getNavBar(id)
     }),
   getNotification: protectedProcedure
+    // type of notification is any.
     .input(getNotificationSchema)
     .query(async ({ input }) => {
       const { siteId } = input
       const notification = await getNotification(siteId)
-      return notification
+      return (notification as string) || ""
```

Comment:
once `getNotification` is typesafe we should remove the cast. 

for learning (not needed in this case): generally if you are casting a primitive, might be better to enforce that primitive with its constructor like so:

```ts
return String(notification ?? "")
```